api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -160,19 +160,25 @@ private String createFirecloudUserAndBillingProject(User user) {
     return createFirecloudBillingProject(user);
   }
 
-  private User saveUserWithConflictHandling(User user) {
-    if (user.getGivenName().length() > 80) {
-      throw new BadRequestException("Given Name length exceeds character limit. (80)");
-    }
-    if (user.getFamilyName().length() > 80) {
-      throw new BadRequestException("Family Name length exceeds character limit. (80)");
-    }
-    if (user.getCurrentPosition() != null && user.getCurrentPosition().length() > 255) {
-      throw new BadRequestException("Current Position length exceeds character limit. (255)");
+  private void validateStringLength(String field, String fieldName, Integer max, Integer min) {
```

Comment:
nit: `int` preferred, unless you need the boxed type for some specific reason (doesn't look like it here).

Reply:
I used the boxed type because I needed to use the `equals` method

Reply:
Why not just use `==` ?

Reply:
that just checks null/not null i believe, not actually checking equality

Reply:
CH is right - you could use the primitive `int min`, use `==`, which will do the same thing as `equals` does on full objects.